recipes/astor/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "astor" %}
+{% set version = "0.5" %}
+{% set md5 = "0fa627a7f14e02566e941ec782c03bfc" %}
```

Comment:
I believe we prefer the sha256 (in this case `4abff251d3f03cbe99106cc16e0dc94adc4f88d580c34a717741c7090b76aab9`).